README.md

```diff
@@ -86,6 +86,15 @@ cuGraph provides an auto-renumbering feature, enabled by default, during Graph c
 
 cuGraph is constantly being updatred and improved. Please see the [Transition Guide](TRANSITIONGUIDE.md) if errors are encountered with newer versions
 
+## Graph Sizes and GPU Memory Size
+As a simple rule of thumb, the amount of GPU memory should be about twice the size of the data size.  That gives overhead for the CSV reader and other transform functions.  There are ways around the rule but using smaller data chunks.  

```

Comment:
I'd recommend adding that "This estimate varies depending on the algorithm and the connectivity pattern of the graph."